css/block.css

```diff
@@ -156,6 +158,84 @@ wb-context[closed] > wb-contains{
     overflow: hidden;
 }
 
+
+wb-value[type=number] > input{
```

Comment:
By the way, only the `background-image` will be overridden due to **THE CASCADE** and specificity rules. I might be on Slack tonight to explain things, betterly!

### Resources
- [What is Specificity](http://www.smashingmagazine.com/2007/07/27/css-specificity-things-you-should-know/)
- [Specificity Calculator](http://specificity.keegan.st/)